packages/react-components/src/index.css

```diff
@@ -1,16 +1,13 @@
-:root {
-    --o-ui-disabled-opacity: 0.4;
-    --o-ui-pill-border-radius: 999px;
-    --o-ui-rounded-border-radius: var(--scale-alpha);
-    --o-ui-circular-border-radius: 100%;
-}
-
 [class^="o-ui-"] {
     --o-ui-focus-ring-inset-sm: 0 0 0 1px var(--o-ui-focus-ring-color);
```

Comment:
Are these var still used since you updated how the focus ring is done?

Reply:
`--o-ui-rounded-border-radius` is still used for inner focus-rings (the ones that are not offset)